components/quick_input/clearable_tooltip.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Tooltip} from 'react-bootstrap';
+import OverlayTrigger from 'components/overlay_trigger';
+import Constants from 'utils/constants.jsx';
+
+interface IClearableTooltipProps{
```

Comment:
nit: empty space between the name and bracket, I saw this in several places, would you be able to use our eslint configuration?